migration.json

```diff
@@ -1260,13 +1260,13 @@
         "Agreement": "0x93635040F665F54a714176473B2cc4B1119c9354",
         "Auction4Reputation": "0x713086D5c5004A5Cb64622F0a2a263976806A417",
         "Avatar": "0x6754AB736E8EDC00c7d54d359AcEba2DEB6c9785",
-        "Competition": "0x0E81F876Fc086e041552a66b90F30F1cbdFD25D2",
+        "Competition": "0x39Bb989F6B07EFAE2DDac3fE46e8B5B2AC9a9899",
```

Comment:
why this file changed?

Reply:
This is the change we made a few days ago, I just pushed to the same or because I need both fixes in the new version